sql/moz-fx-data-shared-prod/addons_derived/dataset_metadata.yaml

```
@@ -0,0 +1,11 @@
+friendly_name: Addons Derived
+description: |-
+  Derived data used for Addons related analysis.
+dataset_base_acl: derived
+user_facing: true
```

I don't think derived datasets can be user-facing.
```suggestion
user_facing: false
```